app/controllers/follows_controller.rb

```diff
@@ -25,6 +25,28 @@ def show
     end
   end
 
+  def bulk_show
```

Comment:
as this is a new endpoint unauthorized, we should probably rate limit it :-D

Reply:
Since I added the guard clause for current_user like the action above I am less worried about this endpoint now. I also expect that we will be bringing down our overall throttle once this is merged which will help a lot.